scripts/novaseq/stage/demux-novaseq.batch

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#SBATCH -c 18
+#SBATCH -p dragen # REMOVE AFTER TESTING
+##SBATCH -p gpu # REMOVE AFTER TESTING
```

Comment:
I think it should mimic production as much as possible. Perhaps you could add the other "dragen" and "gpu" options as comments with a comment